### DISCUSSION TITLE
Background but not the content

### DISCUSSION THREAD

**Discussion by marcinsadlik** [yitengjun/ukiyo-js#27] [Q&A]
Hello,

Could you please be so kind to clarify whether it is possible to add parallax efect only to div's background but not to it's content? When div contain further divs or anything else, it's parallaxed along with background...

**Comment by yitengjun** [comment]
In cases like this, it is necessary to markup the parallax image and content to separate them as follows. 
By positioning the parallax image elements absolutely using `position: absolute` or similar techniques, the parallax image can be placed as the background while separating the content elements. This allows you to apply the parallax effect only to the background.

```html
<div class="background"></div>
<div class="contents">
  <p>contents</p>
</div>
```

**Comment by marcinsadlik** [comment]
I figured out this solution, thought maybe there's something more useable, but stil many thanks for your kind answer! Keep doing good job.
